jscomp/bsb/bsb_theme_init.ml

```diff
@@ -114,7 +114,7 @@ let init_sample_project ~cwd ~theme name =
       else 
         begin
           Format.fprintf Format.err_formatter 
-            "@{<error>Invalid package name@} %S @."
+            "@{<error>Invalid package name@} %S.@}@ The project name must be a valid npm name,@ thus can't contain upper-case letters, for example."
```

Comment:
1. we don't really care about the layout here
2. it seems unbalanced? `@}@`

Reply:
`@}` is right here. I close the error tag. Previously it's force-closed by `@.`

Reply:
The second `@ ` is for the space and break hint